Header header

bool pos_valid

float64[3] pos_ecef
float64[3] pos_llnh
float64 pos_accuracy

bool vel_valid
float64[3] vel_ecef   
float64 vel_accuracy 

int32 num_satellites

